src/components/calculator/calculator.js

```diff
@@ -1,69 +1,62 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import './calculator.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly remove the lines that disable `Eslint` since it's a requirement to follow linter rules to ensure the code is using best practices 👍 Please refer to [this](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md)